redbot/cogs/streams/streams.py

```diff
@@ -226,16 +230,14 @@ async def check_online(
         except InvalidTwitchCredentials:
             await ctx.send(
                 _(
-                    "The Twitch token is either invalid or has not been set. See "
-                    "`{prefix}streamset twitchtoken`."
-                ).format(prefix=ctx.clean_prefix)
+                    "The Twitch token is either invalid or has not been set. See " "{command}."
+                ).format(command=f"`{ctx.clean_prefix}streamset twitchtoken`")
             )
         except InvalidYoutubeCredentials:
             await ctx.send(
                 _(
-                    "The YouTube API key is either invalid or has not been set. See "
-                    "`{prefix}streamset youtubekey`."
-                ).format(prefix=ctx.clean_prefix)
+                    "The YouTube API key is either invalid or has not been set. See " "{command}."
```

Comment:
Could you join these strings? Black doesn't do it automagically right now.
```suggestion
                    "The YouTube API key is either invalid or has not been set. See {command}."
```